src/types/token.ts

```diff
@@ -0,0 +1,30 @@
+// External Libraries
+import * as _ from "lodash";
+
+// Utils
+import { BigNumber } from "../../utils/bignumber";
+import { TOKEN_REGISTRY_TRACKED_TOKENS } from "../../utils/constants";
+
+export class Token {
+    public readonly address: string;
+    public readonly decimals: BigNumber;
```

Comment:
I think this should be standardized as `numDecimals`, to be consistent with the parameters for `TokenAmount`